app/models/manageiq/providers/vmware/manager_auth_mixin.rb

```diff
@@ -17,7 +18,14 @@ def verify_credentials(auth_type = nil, options = {})
           _log.error("Error Class=#{err.class.name}, Message=#{err.message}")
           raise miq_exception
         end
-      when 'amqp' then verify_amqp_credentials(options)
```

Comment:
Now that the two verifications are almost identical, I wonder if you could change the order of the code, i.e.

```
being
  case auth_type.to_s
    when 'default' then
      ..
    when 'amqp' then 
      ..
    else
      ..
  end
rescue ...
```